notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "JAPoU8Sm5E6e"
```

Comment:
<p>This cell has 2 headings? Reduce to one heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Updated.